CMakeLists.txt

```diff
@@ -78,9 +78,19 @@ if(LINUX)
 endif()
 
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${cxx_flags}")
-set(CMAKE_CXX_FLAGS_DEBUG "-g")
+set(CMAKE_CXX_FLAGS_DEBUG "-g -Og -fno-omit-frame-pointer")
```

Comment:
if you need this for ASAN builds only, you could use the `postPatch` hook to edit this file, only for ASAN builds. You are already editing the CMakeLists.txt at line 94 below; we can extend this with some ASAN-specific text.

Reply:
pulling this back! Your change is perfect as it is; ignore this one please

Reply:
I'll move `-fno-omit-frame-pointer` to asan flags, and remove `-Og` as `_FORTIFY_SOURCE` isn't compatible with asan anyway, disable `_FORTIFY_SOURCE` when using asan is the recommend way